index.js

```diff
@@ -3,7 +3,7 @@ const app = express()
 const port = 3000
 
 app.get('/', (req, res) => {
-  res.send('Hello World!')
+  res.send('Hello World 2!')
```

Comment:
why you have use! in your code, not a blocker as of now. But i was curious for this